sequencingSourceFileDetails.md

```diff
@@ -0,0 +1,35 @@
+# Sequencing Source File Details
+
+Poseidon 2.7.0 added an option to specify sequencing source data. This is a tab-separated table, much like the Janno file, but following [a different schema](https://github.com/poseidon-framework/poseidon2-schema/blob/master/sequencingSourceFile_columns.tsv). Note that the primary entities in this table are Sequencing entities (typically corresponding to DNA libraries or even multiple runs/lanes of the same library). The link to the Individuals listed in the Janno-file are made through a foreign-key relationship into `Poseidon_ID`.
```

Comment:
The table defining the columns could be added here or on the `Package definition` page with the code @dhananjaya93 is currently writing :+1:

Reply:
Yes, I agree. I briefly thought whether I should hard-code them into here now, but I think the link to the schema is enough for now. But once Danas code is working, it would be great to include an automatically formatted table here!